homeassistant/components/panasonic_viera/__init__.py

```diff
@@ -219,6 +220,7 @@ async def async_turn_off(self):
         """Turn off the TV."""
         if self.state != STATE_OFF:
             await self.async_send_key(Keys.power)
+            self.state = STATE_OFF
```

Comment:
this line was so that I could use a service call to turn the entity off in the `test_onoff` function. it shouldn't impact functionality, i think

Reply:
How will the state be updated otherwise?

Reply:
if the update method failed ;)